include/lsst/afw/table/Schema.h

```diff
@@ -46,7 +46,7 @@ class Schema {
 public:
 
     // This variable is defined in SchemaImpl, but is replicated here as
-    // a static so that it is available to SWIG.
+    // a static so that it is available to Python.
     static int const VERSION = detail::SchemaImpl::VERSION;
```

Comment:
It should clearly be `static` (i.e., be a class-level constant), but perhaps the comment is clearer if "as a static" is removed from the text.